README.md

```diff
@@ -22,28 +24,33 @@ AutoML的部署脚本维护在``./deploy``目录下.
 - Docker
 - Minikube
 
-安装教程:
+如果没有，参考如下安装教程:
 
 **Mac OS:**
 
-- Docker: https://docs.docker.com/desktop/install/mac-install/
```

Comment:
Perhaps we'd better keep the installation guide of Docker and Minikube

Reply:
I don't think it's necessary

Reply:
Not all contributors in our project are software engineers. There are perhaps also algorithm engineers and UI designers as well as product managers, do you think that whether we need to keep installation guide of Docker and Minikube for them?

Reply:
have been added